tests/test_var_sklearn.py

```diff
@@ -1,74 +1,71 @@
 # Released under The MIT License (MIT)
 # http://opensource.org/licenses/MIT
-# Copyright (c) 2013-2014 SCoT Development Team
+# Copyright (c) 2013-2015 SCoT Development Team
 
 import unittest
 
+from numpy.testing.utils import assert_array_almost_equal
+from numpy.testing.utils import assert_almost_equal
+
 import numpy as np
 from sklearn.linear_model import Ridge, RidgeCV, Lasso, LassoLars, ElasticNet
 
 from scot.backend_sklearn import VAR
 
 
-class TestVAR(unittest.TestCase):
+class CommonTests(unittest.TestCase):
     def setUp(self):
         np.random.seed(12345)
         self.var0 = VAR(2)
         self.var0.coef = np.array([[0.2, 0.1, 0.4, -0.1], [0.3, -0.2, 0.1, 0]])
         self.x = self.var0.simulate((1000, 100))
+        self.var = VAR(2)
 
     def tearDown(self):
         pass
 
     def test_fit(self):
-        var = VAR(2)
+        var = self.var
```

Comment:
I don't know, I think creating an alias here is not really necessary because `self.var` is not that hard to read, and if you also create a local `var` then we have too many names with `var` floating around. So I'd stick to `self.var`.